rmg.py

```diff
@@ -83,7 +83,7 @@ def parse_command_line_arguments(command_line_args=None):
                         metavar='DD:HH:MM:SS', help='set the maximum execution time')
 
     # Add option to select max number of processes for reaction generation
-    parser.add_argument('-n', '--maxProc', type=int, nargs=1, default=1,
+    parser.add_argument('-n', '--maxproc', type=int, nargs=1, default=1,
```

Comment:
It would be ideal if you could put this changes with the prior commit. I can show you how to do that if you'd like.